Networking/NetworkingTests/Responses/wooshipping-get-account-settings-success.json

```
@@ -0,0 +1,50 @@
+{
+   "data":{
+      "storeOptions":{
+         "currency_symbol":"$",
+         "dimension_unit":"cm",
+         "weight_unit":"kg",
+         "origin_country":"US"
+      },
+      "formData":{
+         "selected_payment_method_id":3190997,
+         "enabled":true,
+         "email_receipts":true,
+         "use_last_service":false,
+         "use_last_package":true,
+         "paper_size":"label",
+         "checkout_address_validation":false
+      },
+      "formMeta":{
+         "can_manage_payments":false,
+         "can_edit_settings":true,
+         "master_user_name":"Rachel",
+         "master_user_login":"rachelmcr",
+         "master_user_wpcom_login":"rachelmcr",
+         "master_user_email":"rachel@automattic.com",
```

Let's use fake data in this mock, for example:

```suggestion
         "master_user_name":"John Smith",
         "master_user_login":"jsmith",
         "master_user_wpcom_login":"jsmith",
         "master_user_email":"jsmith@example.com",
```